salt/modules/mount.py

```diff
@@ -291,7 +293,7 @@ def __str__(self):
 
     def __repr__(self):
         '''always works'''
-        return str(self.criteria)
+        return six.text_type(self.criteria)
```

Comment:
I think you and @DmitryKuzmenko may be both working on the same file here? I recall talking through this with him and we found that we actually need to change the `str` to `repr`, not `six.text_type`. Or maybe this is just similar code to what he was working on.

Reply:
hrm...I'm not sure. i'll just fix this here and i can rebase if needed.